src/ipc/Messages.h

```diff
@@ -17,7 +17,7 @@ namespace Ipc
 {
 
 /// message class identifier
-typedef enum { mtNone = 0, mtRegistration,
+typedef enum { mtNone = 0, mtRegistration, mtForegroundRebuild, mtStrandSearchPause,
```

Comment:
I did not find documentation for mtStrandSearchPause. When there is no class where these details can be documented, we should probably do it here while reformatting to have one type per line:

```suggestion
typedef enum {
    mtNone = 0,
    mtRegistration,
    mtForegroundRebuild,
    mtStrandSearchPause, ///< the disker exists but is still building its index
```